features/GGG_CFE_Integration_tests_EmploymentTests/GGG_EMP_1-v5.feature

```diff
@@ -0,0 +1,33 @@
+Feature:
+  "Employed Multiple Incomes test"
+
+  Scenario: Test that the correct output is produced for the following set of data.
+    Given I am undertaking a certificated assessment
+    And A submission date of "2022-01-12"
+    And I add the following proceeding types in the current assessment:
+      | ccms_code | client_involvement_type |
+      | DA001     | A                       |
+      | SE013     | A                       |
+    And I have a dependant aged 2
+    And I add employment income of 2550.33 per month with 0.0 benefits_in_kind, 745.31 tax and 144.06 national insurance
+    And I add the following employment details:
+      | client_id |     date     |  gross   | benefits_in_kind |   tax  | national_insurance  |
+      |     C     |  2021-12-07  |   250.00 |      0           |   80.0 |          0.0        |
+      |     C     |  2021-12-14  |   250.00 |      0           |   80.0 |          0.0        |
```

Comment:
It's annoying how we have to add employment income in 2 different ways, but this works.

Could you not do it like you did on `features/GGG_CFE_Integration_tests_EmploymentTests/GGG_EMP_4-v5.feature`?

Reply:
fair enough, that makes sense.